chef.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "mixlib-shellout", ">= 2.0.0.rc.0", "< 3.0"
   s.add_dependency "ohai", "~> 8.0"
 
-  s.add_dependency "ffi-yajl", ">= 1.2", "< 3.0"
```

Comment:
Aren't there a bunch of other places we could update this?  Off the top of my head there is the [chefdk](https://github.com/chef/chef-dk/blob/master/chef-dk.gemspec#L42).  It would be nice to have as few ffi-yajl versions in our omnibus packages as possible.